admin/src/ConflictDetectionReporter.js

```diff
@@ -26,36 +26,80 @@ const STATUS = {
 const STYLES = {
   container: {
     position: 'fixed',
-    fontFamily: 'sans-serif',
+    fontFamily: 'fa5-proxima-nova,"Helvetica Neue",Helvetica,Arial,sans-serif',
```

Comment:
@frrrances we're not loading this proxima-nova font anywhere in this plugin (unless you added it?) So I think the only viewers who will see it in this typeface will be those who happen to have it installed in their local desktop font manager, which is probably just you, Dave and Brian ;-) Is that what you expect?

Reply:
Good catch - I'm going to leave the rest of the stack, but remove that super special one... thanks.